apps/web/src/components/profile/interval-selector.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+
+import * as React from "react";
+
+import {Select, SelectContent, SelectItem, SelectTrigger, SelectValue} from "@energyleaf/ui";
+
+export default function IntervalSelector({field}) {
```

Comment:
Hier wäre es sinnvoll, den Typen von field noch zu definieren. Außerdem kannst du Field hier auch direkt übergeben, wenn du keine anderen Parameter hast.
```suggestion
export default function IntervalSelector(field) {
```